db/migrate/20200811222853_rename_data_request_note_to_request_type_note_for_data_request.rb

```diff
@@ -0,0 +1,5 @@
+class RenameDataRequestNoteToRequestTypeNoteForDataRequest < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :data_requests, :data_request_note, :request_type_note
+  end
+end
```

Comment:
any particular reason we add "data_request_note" first, then rename it as "request_type_note"?

Reply:
I messed up the name first time round; then I think I had intervening migrations in commits in git. Seemed easier to rename than unpick everything